test/e2e_node/mirror_pod_test.go

```diff
@@ -24,25 +24,21 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/errors"
-	"k8s.io/kubernetes/pkg/client/restclient"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/types"
 	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/test/e2e/framework"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("MirrorPod", func() {
-	var cl *client.Client
-	BeforeEach(func() {
-		// Setup the apiserver client
-		cl = client.NewOrDie(&restclient.Config{Host: *apiServerAddress})
-	})
-	ns := "mirror-pod"
+var _ = framework.KubeDescribe("MirrorPod", func() {
+	f := NewDefaultFramework("mirror-pod")
 	Context("when create a mirror pod ", func() {
 		var staticPodName, mirrorPodName string
 		BeforeEach(func() {
+			ns := f.Namespace.Name
```

Comment:
We need utility functions for creating pods in general in the node e2e suite. This PR doesn't do that, unfortunately.